src/App.jsx

```diff
@@ -14,57 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const SelectedGoods = ({ selectedGood, onClear }) => {
```

Comment:
You don't really need to create separate component for that

Reply:
but it looks better with separate components and is easier to read.